pydrad/parse/parse.py

```diff
@@ -336,168 +341,74 @@ def __repr__(self):
 Filename: {self._phy_filename}
 Timestep #: {self._index}"""
 
-    def _read_phy(self):
+    def _read_amr(self):
         """
-        Parse the hydrodynamics results file
+        Parse the adaptive mesh grid file
         """
-        self._phy_data = np.loadtxt(self._phy_filename)
+        self._amr_data = read_amr_file(self._amr_filename)
 
-    def _read_amr(self):
+    def _read_phy(self):
         """
-        Parse the adaptive mesh grid file
+        Parse the hydrodynamics results file
         """
-        # TODO: Read other quantities from .amr file
-        with self._amr_filename.open() as f:
-            lines = f.readlines()
-            self._grid_centers = np.zeros((int(lines[3]),))
-            self._grid_widths = np.zeros((int(lines[3]),))
-            for i, l in enumerate(lines[4:]):
-                tmp = np.array(l.split(), dtype=float)
-                self._grid_centers[i] = tmp[0]
-                self._grid_widths[i] = tmp[1]
+        if not self._phy_filename.is_file():
+            log.warning(f'{self._phy_filename} not found. Skipping parsing of phy files. Set read_phy=False to suppress this warning.')
+            return
+        self._phy_data = read_phy_file(self._phy_filename)
+        # NOTE: only adding three columns in this manner as the remaining columns are dealt with explicitly
+        # below because of the overlap with the derived quantities from the .amr files.
```

Comment:
Not sure I understand this -- does this mean that the loop coordinate s, for example, is pulled from the .amr files and therefore not parsed from the .phy files?

That seems like a lot of overhead.  For most plots, we want the data from the .phy files first and foremost.  I would think it make sense to do the inverse of this -- fully parse the .phy files and only save the excess from the .amr files, which are also bigger in filesize.  

I guess the counter-argument is that the .amr files should always be printed, and the .phy files are optional?  But in practice would we ever actually run without printing out the .phy files?

Reply:
I started down this path of deprioritizing the `phy` files is because @sjbradshaw mentioned he has considered removing them entirely from the output as you can just derive all of those quantities from the output in the `amr` files.

Additionally, if you're running *many* HYDRAD simulations, you might want to avoid outputting the `phy` files to minimize the number of files produced per run. As you say, you always have to produce the `amr` files.

> Not sure I understand this -- does this mean that the loop coordinate s, for example, is pulled from the .amr files and therefore not parsed from the .phy files?

That is correct. 

> I guess the counter-argument is that the .amr files should always be printed, and the .phy files are optional? But in practice would we ever actually run without printing out the .phy files?

That's my thinking. Currently, you have to read both the `amr` and `phy` files to get all the coordinate info and the thermodynamic variables. With this PR, you can get all of that by just reading the `amr` files so you are having to parse one less file.